exporter/otlp/opentelemetry-exporter-otlp.gemspec

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'opentelemetry/exporter/otlp/version'
+
+Gem::Specification.new do |spec|
+  spec.name        = 'opentelemetry-exporter-otlp'
+  spec.version     = OpenTelemetry::Exporter::OTLP::VERSION
+  spec.authors     = ['OpenTelemetry Authors']
+  spec.email       = ['cncf-opentelemetry-contributors@lists.cncf.io']
+
+  spec.summary     = 'OTLP exporter for the OpenTelemetry framework'
+  spec.description = 'OTLP exporter for the OpenTelemetry framework'
+  spec.homepage    = 'https://github.com/open-telemetry/opentelemetry-ruby'
+  spec.license     = 'Apache-2.0'
+
+  spec.files = ::Dir.glob('lib/**/*.rb') +
+               ::Dir.glob('*.md') +
+               ['LICENSE', '.yardopts']
+  spec.require_paths = ['lib']
+  spec.required_ruby_version = '>= 2.5.0'
+
+  spec.add_dependency 'google-protobuf', '>= 3.4.1.1', '< 4'
```

Comment:
There appears to be some jruby wonkiness with this dependancy, it would seem? Have you had a chance to look into the test failures on jruby?

Reply:
I took a brief look. It seems as though we need to install the Java protobuf package using maven. I’m tempted to not support jruby with this gem for now, but I’m not sure how to do that yet.

Reply:
FWIW, every app I have been using this gem with is running in JRuby

Reply:
So I looked into this a bit, as Datadog supports jRuby and has a GRPC integration (which relies on this gem). And as It turns out we [don't support GRPC instrumentation on jruby](https://github.com/DataDog/dd-trace-rb/pull/1126#issue-457957841) since the google-protobuf gem itself does not support jruby (and grpc relies on this gem). It's impacting a lot of ruby library folks, it would seem. This is the issue in question, for context: https://github.com/protocolbuffers/protobuf/issues/1594

It's a long issue, and I can't say I have fully grok'd things, but tl;dr it sounds like there's not enough free hands right now within jruby/google to add support at the moment.

There might be some viable workarounds on the thread but it's not clear to me, someone on there mentioned a pure ruby implementation: https://github.com/ruby-protobuf/protobuf , but it doesn't seem very actively maintained at the moment. The reasonable path forward here to me would be not include jruby support for now, mostly because I am not clever enough to know how to work around this.

Fwiw there are definitely some large enterprises out there who are both JRuby users and users of tracing products.

Reply:
To be clear, I won’t remove jruby support from OTel Ruby in general, just from the OTLP exporter. The Jaeger exporter should be a viable alternative for jruby shops until we can get protobuf working.

Reply:
Based on that, I guess it's a really good thing https://github.com/open-telemetry/opentelemetry-ruby/issues/231 is already in the upcoming milestone. I think you are right that this should provide a viable alternative and, for me, may actually simplify things since my local dev environments are using local Jaeger instances for validation of trace data.